patch/frontend/app/components/ui/chat/chat-messages.tsx

```diff
@@ -46,7 +46,7 @@ export default function ChatMessages(
   }, [messageLength, lastMessage]);
 
   return (
-    <div className="w-full h-full rounded-xl bg-white p-4 shadow-xl border overflow-auto">
+    <div className="w-full h-full rounded-xl bg-white p-4 shadow-xl border overflow-auto relative">
```

Comment:
@thucpn please talk with @leehuwuj about this `relative` when changing CL upstream and ensure it would also work with an unchanged upstream

Reply:
This styling is already applied in CL with the conversation starters. I didn't add it in the last PR because our patching changed the styling too. I already talked to Thuc before adding this, so `relative` was added in CL to show the starter question correctly. It's probably better if we have a new style that both CL and RAGapp can share.

Reply:
Yes that would be great. @thucpn please look at this tomorrow